requirements.readthedocs.txt

```diff
@@ -55,6 +55,7 @@ bz2file>=0.98,<0.99                               # Read and write bzip2-compres
 python-magic>=0.4.15,<0.5.0                       # File type identification using libmagic
 futures>=3.2.0; python_version < '3.0'            # Clean single-source support for Python 3 and 2
 six>=1.11.0                                       # Python 2 and 3 compatibility utilities
+boto3>=1.9.86					  # S3 boto protocol (new version)
```

Comment:
this also needs to go to the main requirements, right @bari12  ?

Reply:
Correct, this needs to go to the pip-requires-client file. The readthedocs is generated from these.

Reply:
Sorry, have removed this now. It was the only place I could see in the source code that listed the requirements. I didn't realise it was updated automatically.

Reply:
Hi James, no worries. You still have to add it to the `pip-requires-client` file https://github.com/rucio/rucio/blob/master/tools/pip-requires-client
But you don't need to add it specifically to the readthedocs.txt one, there it will be added by the build procedure.

Reply:
Thanks, I have now added it to `pip-requires-client` instead.